src/app/_components/BreadCrumb/BreadCrumb.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { ChevronRightIcon } from "@heroicons/react/20/solid";
+
+interface BreadCrumbProps {
+  items: Array<{
```

Comment:
You can probably create an items type so that you can just call an array of items like this 
type item = {
// what you want the item to have
} then call items[] in you interface instead of what you did here

Reply:
will fix it-2 :)